test/fixtures/if.rb

```diff
@@ -35,3 +35,9 @@
 %
 if foo {}
 end
+%
```

Comment:
Surprised this new fixture [doesn't result in a test failure in CI](https://github.com/ruby-syntax-tree/syntax_tree/actions/runs/2302180574). I get one locally:
```
  1) Failure:
SyntaxTree::FormattingTest#test_formatted_if_9 [/Users/brian.buchalter/workspace/syntax_tree/test/formatting_test.rb:9]:
--- expected
+++ actual
@@ -1,6 +1,2 @@
-"if not true
-  puts('not_condition')
-else
-  puts('condition')
-end
+"not true ? puts(\"not_condition\") : puts(\"condition\")
 "
```